tests/ztest/ztest_param/pytest/test_parameters.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+import pytest
+from twister_harness import  Shell
+
+def test_run_all_shuffled(shell: Shell):
+    shell.exec_command('ztest shuffle -c 1 -s 1')
+
+def test_run_testcase_once(shell: Shell):
+    shell.exec_command('ztest run-testcase ztest_params::test_dummy')
+
+def test_run_testcase_twice(shell: Shell):
+    shell.exec_command('ztest run-testcase ztest_params::test_dummy -r 2')
+
+def test_run_testsuite_twice(shell: Shell):
+    shell.exec_command('ztest run-testsuite ztest_params -r 2')
+
+def test_run_testsuite_once(shell: Shell):
+    shell.exec_command('ztest run-testsuite ztest_params')
+
+def test_run_testcase_with_int_parameter(shell: Shell):
+    lines = shell.exec_command('ztest run-testcase ztest_params::test_int_param -p 44')
+    assert 'Passed int value:44' in lines, 'expected response not found'
+
+def test_run_testcase_with_string_parameter(shell: Shell):
+    lines = shell.exec_command('ztest run-testcase ztest_params::test_string_param -p Zephyr')
+    assert 'Passed string:Zephyr' in lines, 'expected response not found'
```

Comment:
This one seems to be fully redundant, as the next test covers this case; I'd remove it.

Reply:
It shows how to use `-p` with one parameter, but as recommended, I've removed it.